.github/workflows/ci.yml

```diff
@@ -111,18 +111,44 @@ jobs:
           python -m pip install -r requirements-ci.txt
       - name: analyze example session
         timeout-minutes: 5
+        if: always()
         run: |
           . testenv/bin/activate
           ./examples/00_getting_started.py local.localhost
-          SID=$(ls -rt | grep rp.session)
-          echo "$SID: $SID"
+          SID=$(find . -name "rp.session*" -type d -exec basename {} "rp.session" \;)
+          echo "SID analyse example session: $SID"
           radical-analytics-inspect "$SID"
           mkdir artifacts
           ls -la
           cp -R *.png *.stats artifacts
+      - name: Collect session
+        if: always()
+        run: |
+          SIDCLIENT=$(find . -name "rp.session*" -type d -exec basename {} "rp.session" \;)
+          echo "SIDCLIENT collect session : $SIDCLIENT"
+          SIDAGENT="$HOME/radical.pilot.sandbox/$SIDCLIENT"
+          CLIENTNAME="analytics_client_$SIDCLIENT"
+          AGENTNAME="analytics_agent_$SIDCLIENT"
+          mkdir sessions_analytics
+          # tar cvfj $CLIENTNAME.tar.bz2 $SIDCLIENT
+          tar cvfj $CLIENTNAME.tar.bz2 `find $SIDCLIENT -type d -print`
```

Comment:
try to change the order of the options -> `cvjf` (that is something tricky with keeping `f` the last one - just discovered it)

Reply:
that same would be for tutorials

Reply:
Yeah, tar should need the `f` option last as that takes the `F`ilename argument which follows the set of options.